test/stdlib/Diffing.swift

```diff
@@ -0,0 +1,667 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import Swift
+import StdlibUnittest
+
+let suite = TestSuite("Diffing")
+
+suite.test("Diffing empty collections") {
+  let a = [Int]()
+  let b = [Int]()
+  let diff = b.difference(from: a)
+  expectEqual(diff, a.difference(from: a))
+  expectTrue(diff.isEmpty)
+}
+
+suite.test("Basic diffing algorithm validators") {
+  let expectedChanges: [(
+    source: [String],
+    target: [String],
+    changes: [CollectionDifference<String>.Change],
```

Comment:
Oh.. So we don't need availability checks if the APIs are marked as 9999?

Reply:
Ah, good catch -- I haven't run the tests after the availability commit.

Reply:
I still have to guard for it, but it will succeed at runtime